charts/astronomer/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: astronomer
-version: 0.18.0
+version: 0.18.1
```

This is not necessary because 0.18.0 is not released https://github.com/astronomer/astronomer/releases
```suggestion
version: 0.18.0
```